samples/dependency-injection/extensions-dependency-injection/Extensions.DependencyInjection_1/Sample/Program.cs

```diff
@@ -12,24 +12,31 @@ static async Task Main()
         #region ContainerConfiguration
 
         var endpointConfiguration = new EndpointConfiguration("Sample");
+        endpointConfiguration.UseTransport<LearningTransport>();
+
+        var serviceCollection = new ServiceCollection();
```

Comment:
would it make sense to move all ServiceCollection related code before the endpoint configuration, so that the endpoint configuration is not "interrupted" by ServiceCollection configuration?

Reply:
have a look

Reply:
👍